Directory.Build.targets

```diff
@@ -157,7 +157,7 @@
   </PropertyGroup>
 
   <!-- Properties for Package Validation -->
-  <PropertyGroup Condition="'$(ExcludeFromSourceBuild)' != 'true'">
+  <PropertyGroup Condition="'$(ExcludeFromSourceOnlyBuild)' != 'true'">
```

Comment:
Much of these updates are just straight replacements for source-only scenarios. Microsoft's build of the VMR doesn't use SourceOnly.

Reply:
What's the difference between `ExcludeFromSourceBuild` and `ExcludeFromSourceOnlyBuild`?

Reply:
The old control name was ExcludeFromSourceBuild. new control name is ExcludeFromSourceOnlyBuild. They do the same thing.